### PR TITLE
cloud: add security FAQ

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/tidb-cloud-security-faq.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+- General Data Protection Regulation (GDPR)
+
+For more information, see [PingCAP Trust & Compliance Center](https://en.pingcap.com/trust-compliance-center).
+
+## How does TiDB Cloud ensure the security of data transfer between client applications and TiDB clusters?
+
+TiDB Cloud provides the encryption ability of data transfer between client applications and TiDB clusters as follows:
+
+- TiDB Cloud supports [VPC Peering](/tidb-cloud/set-up-vpc-peering-connections.md), which ensures the security of the data transfer between applications and TiDB cluster.
+- For the connection to a TiDB Cloud cluster via a MySQL client, TiDB Cloud enables data transfer encryption by default.
```
Comment (ljun0712) [inline tidb-cloud/tidb-cloud-security-faq.md:26]: For the connection to a TiDB Cloud cluster via a MySQL client, TiDB Cloud cluster enables data transfer encryption by default.

Comment (ljun0712) [inline tidb-cloud/tidb-cloud-security-faq.md:26]:
```suggestion
- For the connection to a TiDB Cloud cluster via a MySQL client, TiDB Cloud cluster enables data transfer encryption by default.
```

File: tidb-cloud/tidb-cloud-security-faq.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+TiDB Cloud supports SSO login using Google or GitHub accounts.
+
+## How do I connect to a cluster in TiDB Cloud?
+
+To connect to a cluster in TiDB Cloud, you can use one of the following methods:
+
+- Connect via standard connection, which allows you to create [traffic filters](/tidb-cloud/tidb-cloud-glossary.md#traffic-filter) for the cluster to secure user access.
+- Connect via VPC peering connection, which is a networking connection between two VPCs that enables you to route traffic between them using private IP addresses.
+- Connect via web SQL shell in the TiDB Cloud console
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-security-faq.md:45]:
```suggestion
- Connect via web SQL shell in the TiDB Cloud console.
```

**Comment by qiancai** [comment]
Close this PR because there is another [PR](https://github.com/pingcap/docs/pull/12498/files) for security FAQs

**Pull request closed without merging**